presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -915,91 +1018,6 @@ Drop the schema ``iceberg.web``::
 
     DROP SCHEMA iceberg.web
 
-Register table
-^^^^^^^^^^^^^^
-
-Iceberg tables for which table data and metadata already exist in the
-file system can be registered with the catalog using the ``register_table``
-procedure on the catalog's ``system`` schema by supplying the target schema,
-desired table name, and the location of the table metadata::
-
-    CALL iceberg.system.register_table('schema_name', 'table_name', 'hdfs://localhost:9000/path/to/iceberg/table/metadata/dir')
-
-.. note::
-
-    If multiple metadata files of the same version exist at the specified
-    location, the most recently modified one will be used.
-
-A metadata file can optionally be included as an argument to ``register_table``
-in the case where a specific metadata file contains the targeted table state::
-
-    CALL iceberg.system.register_table('schema_name', 'table_name', 'hdfs://localhost:9000/path/to/iceberg/table/metadata/dir', '00000-35a08aed-f4b0-4010-95d2-9d73ef4be01c.metadata.json')
-
-.. note::
-
-    The Iceberg REST catalog may not support table register depending on the
-    type of the backing catalog.
```

Comment:
I think we shouldn't simply discard the content added last time to resolve the conflict. This content is valid and should be added to the content you moved above.

Reply:
added.